TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgSectionSymbol.cs

```diff
@@ -0,0 +1,42 @@
+﻿using TypeCobol.Compiler.Scopes;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// A Section symbol used by Cfg : it contains Sentences and Paragraphs in order of appearance.
+        /// </summary>
+        internal class CfgSectionSymbol : SectionSymbol
```

Comment:
Same remark as for `CfgParagraphSymbol`.

Reply:
Same remark as for CfgParagraphSymbol.

Reply:
See PR #1738.